src/styles/main.scss

```
@@ -1 +1,47 @@
-// put code here
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  background-color: white;
+}
+
+.watch {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  border: 2px solid #333;
+  border-radius: 50%;
+  background-color: bisque;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.arrow {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform-origin: 50% 100%;
+  transform: translateX(-50%) translateY(-100%) rotate(0deg);
+  animation-timing-function: steps(60);
+
+  &--minute {
+    width: 3px;
+    height: 80px;
+    background-color: black;
+    animation: rotate 3600s linear infinite;
```

you can move this timing to the variable
```suggestion
    animation: rotate ... linear infinite;
```